### PR TITLE
Create divyashaktipal.json

### PR DESCRIPTION
**Pull request by divyashaktipal**
My social media profile

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6478"></a>

### CODE REVIEW

File: data/divyashaktipal.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Divyashakti Pal",
+  "type": "personal",
+  "bio": "Student "
```
Comment (jaydeepkhatri) [inline data/divyashaktipal.json:4]: The JSON is invalid, You need to add a comment after "bio" property.

```json
"bio": "student",
```

**Review by jaydeepkhatri — CHANGES_REQUESTED** [review]
Also found the spelling for `Linkedin` & `Subscribe` are wrong.

**Review by jaydeepkhatri — APPROVED** [review]
Everything LGTM 🥳

**Review by SaraJaoude — APPROVED** [review]
looks good

File: data/divyashaktipal.json (hunk 1 above)
Comment (SaraJaoude) [reply data/divyashaktipal.json]: I have corrected this

**Pull request merged**